servatrice/src/serversocketinterface.cpp

```diff
@@ -863,6 +895,9 @@ Response::ResponseCode ServerSocketInterface::cmdRegisterAccount(const Command_R
         return Response::RespUsernameInvalid;
     }
 
+    if (userName.toLower() == "servatrice")
+        return Response::RespUsernameInvalid;
```

Comment:
Spaces aren't allowed in usernames by default, but I do agree it could be trimmed here.